app/controllers/credit_mutations_controller.rb

```diff
@@ -17,7 +17,7 @@ def create # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
 
     respond_to do |format|
       if @mutation.save
-        NewCreditMutationNotificationJob.perform_later(@mutation) if Rails.env.production? || Rails.env.staging?
+        NewCreditMutationNotificationJob.perform_later(@mutation) if Rails.env.production? || Rails.env.staging? || Rails.env.luxproduction?
```

Comment:
With this amount of environments, maybe here and on other places it's nicer to do something like `unless Rails.env.development?` (not sure if that's correct ruby)